src/pydata_sphinx_theme/__init__.py

```diff
@@ -501,6 +502,50 @@ def get_edit_url():
     context["theme_show_toc_level"] = int(context.get("theme_show_toc_level", 1))
 
 
+# -----------------------------------------------------------------------------
```

Comment:
Could you look at [the Furo colors documentation](https://github.com/pradyunsg/furo/blob/d0f880da9453d6a5c2363b9b5c24927ed0c2747f/docs/customisation/colors.md) for how to handle Pygments? 

He also has a lot of logic about this [in the `__init__.py` module](https://github.com/pradyunsg/furo/blob/9a1190116ac35e96f4311b5411b9c863f2ad85ac/src/furo/__init__.py) that might provide inspiration.

How does this implementation compare?

Reply:
He did more or less the same. there are some extra steps for the CSS selection and the loading priority. The huge difference is this big hack (https://github.com/pradyunsg/furo/blob/d0f880da9453d6a5c2363b9b5c24927ed0c2747f/src/furo/__init__.py#L167) to read the `pygments_dark_style ` parameter from html_config. I'm not sure it's worth the hack and I'll prefer to use a theme-related configuration variable as you suggested.